netmiko/alcatel/alcatel_aos_ssh.py

```diff
@@ -0,0 +1,42 @@
+"""Alcatel-Lucent Enterprise AOS support."""
+from __future__ import print_function
+from __future__ import unicode_literals
+from netmiko.cisco_base_connection import CiscoSSHConnection
+
+
+class AlcatelAosSSH(CiscoSSHConnection):
+    """Alcatel-Lucent Enterprise AOS support."""
+    def __init__(self, *args, **kwargs):
+        super(AlcatelAosSSH, self).__init__(*args, **kwargs)
+        self._config_mode = False
```

Comment:
General pattern in Netmiko is not to emulate config mode (if it doesn't exist). So I think we should get rid of the `self._config_mode` variable and not track this.